src/Core/Utility.js

```diff
@@ -207,6 +207,16 @@ function hash (str) {
 	return code >>> 0
 }
 
+/**
+ * @param {function} callback
+ * @return {number}
+ */
+function timeout (callback) {
+	return setTimeout(function () {
+		callback(now())
+	}, 16)
```

Comment:
Maybe 17 ms as 1000 / 60 is 16.7

Reply:
`16ms = 62.5fps`
`17ms = 58.8fps`

standard refresh rate is `60Hz`
so 16ms maximizes frames

Reply:
I just thought it was intended as a polyfill for `requestAnimationFrame`, not something to maximize fps, one could use 1ms to get 1000 fps if the goal was to maximize fps. Although, maybe 16ms is better as there will be delay due to executing JavaScript anyways.

Reply:
Above 60fps is worthless because of the `60Hz` refresh rate (i.e. why `1ms` doesn't make sense), but below 60fps your missing frames

Reply:
I don't think above `60Hz` is worthless, firstly, because not all people have 60Hz monitor refresh rate, some have higher than that. Also, even if you have only 60Hz refresh rate, the more fps you generate the smoother the animation will feel.

Reply:
I was unable to find statistics on monitor refresh rates, so lets estimate:
desktop web market share is [~44%](http://gs.statcounter.com/platform-market-share/desktop-mobile-tablet) (We, safely, assume mobile devices are `60Hz`).
Of desktops [~71% (67% + 4%)](http://gs.statcounter.com/browser-market-share/desktop/worldwide) run Chrome/Edge, which auto-update, and thus are likely to include `requestAnimationFrame`.

So, given `requestAnimationFrame` does not exist, we estimate at most `.44 * (1-.71) ~= 13%` of users will ever hit this function.
Given a user is using and older browser, I'd guess the likelihood of their computer having a >60Hz monitor is 1%.
So I'd estimate the total users who could ever use more than `60Hz` is `0.1%`

As for your comment about animation smoothness, it's simply false, because physics. Not just that, it can create artifacts called [`screen tearing`](https://en.wikipedia.org/wiki/Screen_tearing). You are perhaps conflating physics updates with draw updates, which should run at separate rates to optimize animation smoothness / input lag - [Game Loop](http://gameprogrammingpatterns.com/game-loop.html)

Reply:
> As for your comment about animation smoothness, it's simply false, because physics. 

It's actually true, exactly because of physics. Have you ever played computer games?

> At 60 fps, your video card is making a new frame every 1/60 of a second or every 16.67 ms. That means every frame that your monitor shows can be delayed up to 16.67 ms.
>
>At 300 fps, your video card is making a new frame every 1/300 of a second or every 3.33 ms. That means every frame that your monitor shows can only be delayed up to 3.33 ms.
>
> Basically, at 300 fps, each frame your monitor shows is more accurate and less delayed, even though it is only showing 60 frames per second.

Reply:
Another thing is, if you will use that function to create a [DRAF](https://stackoverflow.com/questions/44145740/how-does-double-requestanimationframe-work), it will actually be 1.3 ms too short.

Reply:
I think I understand what you're saying. I now agree that an increase in fps can lead to increased smoothness.

During my research I found this gem: [Rendering for "Glass Time"](https://docs.google.com/presentation/d/1oKEunkaeiTwznGaIX_yIhe6HPfZBXhtv8r5J5hz52UI/edit), which I found fascinating.

For posterity, the official shim: https://gist.github.com/paulirish/1579671

Reply:
The "Glass Time" presentation is indeed interesting. That's why even when you have 60Hz screen its nice to have 300fps when playing computer games.